extensions-core/azure-extensions/src/test/java/org/apache/druid/storage/azure/AzureStorageDruidModuleTest.java

```diff
@@ -303,6 +303,25 @@ public void testBothAccountKeyAndSAStokenUnset()
     );
   }
 
+  @Test
+  public void testGetAzureUtilsWithDefaultProperties()
+  {
+    Properties properties = initializePropertes();
+    AzureAccountConfig config = makeInjectorWithProperties(properties).getInstance(AzureAccountConfig.class);
+    String outputBlob = AzureUtils.maybeRemoveAzurePathPrefix("blob.core.windows.net/container/blob", config.getBlobStorageEndpoint());
+    Assert.assertEquals("container/blob", outputBlob);
+  }
+
+  @Test
+  public void testGetAzureUtilsWithDefaultCustomBlobPath()
+  {
+    Properties properties = initializePropertes();
+    properties.setProperty("druid.azure.endpointSuffix", "core.usgovcloudapi.net");
+    AzureAccountConfig config = makeInjectorWithProperties(properties).getInstance(AzureAccountConfig.class);
+    String outputBlob = AzureUtils.maybeRemoveAzurePathPrefix("blob.core.usgovcloudapi.net/container/blob", config.getBlobStorageEndpoint());
+    Assert.assertEquals("container/blob", outputBlob);
```

Comment:
These tests should move to AzureUtilsTest. Here you can test that AzureAccountConfig is correctly initialized with the properties you are setting. And in AzureUtilsTest, you will add these tests but without creating an AzureAccountConfig object.

Reply:
Done